docs/architecture/dapr-for-net-developers/dapr-at-20000-feet.md

```diff
@@ -0,0 +1,257 @@
+| [Observability](https://github.com/dapr/components-contrib/tree/master/bindings) | Monitor and measure message calls across networked services. |
+| [Secrets](https://github.com/dapr/components-contrib/tree/master/middleware) | Securely access external secret stores. |
+| [Actors](https://github.com/dapr/components-contrib/tree/master/secretstores) | Encapsulate logic and data in reusable actor objects. |
+| [Resource bindings and triggers](https://github.com/dapr/components-contrib/tree/master/exporters) | Trigger code from events raised by external resources with bi-directional communication. |
+
+Building blocks abstract the implementation of distributed application capabilities from your services. Figure 2-3 shows this interaction.
+
+![Dapr building blocks](./media/building-block-integration.png)
+
+**Figure 2-3**. Dapr building block integration.
+
+Under the hood, building blocks invoke Dapr components that provide the concrete implementation for each resource. The code for your service is only aware of the building block. It takes no dependencies on external SDKs or libraries - Dapr handles the plumbing for you. Each building block is independent. You can use one, some, or all of them in your application. As a value-add, Dapr building blocks bake in industry best practices.
+
+We provide detailed explanation and code samples for each Dapr building block in the upcoming chapters. At this point, our jet descends even more. From our new perspective, we can now have a closer look at the Dapr components layer.
+
+### Components
+
+While building blocks expose an API to invoke distributed application capabilities, Dapr components provide the concrete implementation to make it happen.
+
+Consider, the Dapr **state store** component. It provides a uniform way to manage state for CRUD operations. Without any change to your service code, you could switch between any of the following Dapr state components:
+
+- AWS DynamoDB
+- Aerospike
+- Azure Blob Storage
+- Azure CosmosDB
+- Azure Table Storage
+- Cassandra
+- Cloud Firestore (Datastore mode)
+- CloudState
+- Couchbase
+- Etcd
+- HashiCorp Consul
+- Hazelcast
+- Memcached
+- MongoDB
+- PostgreSQL
+- Redis
+- RethinkDB
+- SQL Server
+- Zookeeper
+
+Each component provides the necessary implementation through a common state management interface:
+
+   ```go
+   type Store interface {
+     Init(metadata Metadata) error
+     Delete(req *DeleteRequest) error
+     BulkDelete(req []DeleteRequest) error
+     Get(req *GetRequest) (*GetResponse, error)
+     Set(req *SetRequest) error
+     BulkSet(req []SetRequest) error
+  }
+   ```
+
+   > Note: The Dapr interface above along with all of Dapr has been written in the Golang, or Go, platform. Go is a popular language across the open source community and attests to cross-platform commitment of Dapr.
```

Comment:
```suggestion
> [!TIP]
> The Dapr interface above along with all of Dapr has been written in the Golang, or Go, platform. Go is a popular language across the open source community and attests to cross-platform commitment of Dapr.
```

Instead of indented quotes, that start with `"Note: "` [please use Alerts](https://docs.microsoft.com/en-us/contribute/markdown-reference#alerts-note-tip-important-caution-warning).